milli/src/update/index_documents/extract/extract_geo_points.rs

```diff
@@ -0,0 +1,49 @@
+use std::fs::File;
+use std::io;
+
+use concat_arrays::concat_arrays;
+use log::warn;
+use serde_json::Value;
+
+use super::helpers::{create_writer, writer_into_reader, GrenadParameters};
+use crate::{FieldId, InternalError, Result};
+
+/// Extracts the geographical coordinates contained in each document under the `_geo` field.
+///
+/// Returns the generated grenad reader containing the docid as key associated to the (latitude, longitude)
+pub fn extract_geo_points<R: io::Read>(
+    mut obkv_documents: grenad::Reader<R>,
+    indexer: GrenadParameters,
+    geo_field_id: Option<FieldId>, // faire un grenad vide
```

Comment:
why don't you call `extract_geo_points` only if a geo_field_id exists?

Reply:
Yep that's smarter actually :joy: